account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,64 @@
+import { custom, parseEther, publicActions } from "viem";
+
+import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
+
+import { createSMAV2AccountClient } from "./client.js";
+
+import { local070InstanceOptSep } from "~test/instances.js";
+import { setBalance } from "viem/actions";
+import { accounts } from "~test/constants.js";
+
+describe("MA v2 Tests", async () => {
+  const instance = local070InstanceOptSep;
+  const client = instance.getClient().extend(publicActions);
```

Comment:
I wouldn't do this here actually. the existing LA tests I think do something they shouldn't be doing.

MultiOwnerLA tests do this at the top:
```
const instance = local070Instance;
  let client: ReturnType<typeof instance.getClient>;

  beforeAll(async () => {
    client = instance.getClient();
  });
```

and then the `givenConnectedProvider` passes in a `transport(client)`. Think I ran into some cases where not doing this sometimes ended up with a different instance in the pool to be used

Reply:
the same happened to me earlier lol, wasn't sure what was happening. glad to know the RCA, will change to this pattern